src/master-playlist-controller.js

```diff
@@ -767,7 +767,13 @@ export class MasterPlaylistController extends videojs.EventTarget {
     // code in video.js but is required because play() must be invoked
     // *after* the media source has opened.
     if (this.tech_.autoplay()) {
-      this.tech_.play();
+      const playPromise = this.tech_.play();
+
+      // Catch/silence error when a pause interrupts a play request
+      // on browsers which return a promise
+      if (playPromise !== undefined && typeof playPromise.then === 'function') {
```

Comment:
I think the linter might warn to use `typeof playPromise !== 'undefined'`

Reply:
@gesinger cool! I'll update later today.